src/Schema/Directives/RenameArgsDirective.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Nuwave\Lighthouse\Schema\Directives;
+
+use Closure;
+use GraphQL\Type\Definition\ResolveInfo;
+use Nuwave\Lighthouse\Schema\Values\FieldValue;
+use Nuwave\Lighthouse\Support\Contracts\ArgDirective;
+use Nuwave\Lighthouse\Support\Contracts\DefinedDirective;
+use Nuwave\Lighthouse\Support\Contracts\FieldMiddleware;
+use Nuwave\Lighthouse\Support\Contracts\GraphQLContext;
+
+class RenameArgsDirective extends BaseDirective implements ArgDirective, FieldMiddleware, DefinedDirective
+{
+    public static function definition(): string
+    {
+        return /** @lang GraphQL */ <<<'SDL'
+"""
+Apply the @rename directives on the incoming arguments.
+"""
+directive @renameArgs on FIELD_DEFINITION
```

Comment:
You use both `@raname` and `@renameArgs` here. I think you mean `@renameArgs` in both cases?

Reply:
That's actually what i meant, even though it seems weird at first. `@renameArgs` is not meant to be used by itself in most cases, it is added implicitly.

It does actually apply the `@rename` directives, which would not do anything by themselves.